code/game/objects/items/weapons/twohanded.dm

```diff
@@ -1127,3 +1128,105 @@
 			return ..()
 
 #undef BROOM_PUSH_LIMIT
+
+/obj/item/twohanded/supermatter  //Supermatter Halberd, used by Oblivion Enforcers
+	name = "supermatter halberd"
+	desc = "The revered weapon of Oblivion Enforcers, used to enforce the Order's will."
+	icon_state = "smhalberd0"
+	force = 5
+	sharp = TRUE
+	damtype = BURN
+	w_class = WEIGHT_CLASS_BULKY
+	slot_flags = SLOT_BACK
+	throwforce = 15
+	force_unwielded = 5
+	force_wielded = 40
+	toolspeed = 0.25
+	attack_verb = list("enlightened", "enforced", "cleaved", "stabbed", "whacked")
+	hitsound = 'sound/weapons/bladeslice.ogg'
+	resistance_flags = FIRE_PROOF
+	var/static/list/obliteration_targets = list(/turf/simulated/wall, /obj/machinery/door/airlock)
+	var/charged = TRUE //Whether we'll knockdown on hit
+
+/obj/item/twohanded/supermatter/Initialize(mapload)
+	. = ..()
+	ADD_TRAIT(src, TRAIT_FORCES_OPEN_DOORS_ITEM, ROUNDSTART_TRAIT)
+	AddComponent(/datum/component/parry, _stamina_constant = 2, _stamina_coefficient = 0.5, _parryable_attack_types = ALL_ATTACK_TYPES)
+	//Not sure on the numbers here (remind me to remove this comment before merge if I don't do it myself)
```

Comment:
still not sure?

Reply:
Still not sure what the numbers do here yeah, I'll look into it soon